README.md

```diff
@@ -1,6 +1,6 @@
 # hadoop-router-fs
 
-[RouterFileSystem](src/main/java/io/lakefs/RouterFileSystem.java) is a Hadoop [FileSystem](https://hadoop.apache.org/docs/stable/api/org/apache/hadoop/fs/FileSystem.html) 
+[RouterFileSystem](src/main/java/io/lakefs/RouterFileSystem.java) is a Hadoop [FileSystem](https://hadoop.apache.org/docs/stable/api/org/apache/hadoop/fs/FileSystem.html) class
```

Comment:
I think I prefer the old one.  Spark users see it as a "FileSystem" they can use.  Hadoop has mixed usages, but usually just calls it "S3AFileSystem" (without "class"). The main place I found that does say "S3AFileSystem class" was in its [configuration guide](https://hadoop.apache.org/docs/stable/hadoop-aws/tools/hadoop-aws/index.html#general_configuration):

> ```xml
> <property>
>   <name>fs.s3a.impl</name>
>   <value>org.apache.hadoop.fs.s3a.S3AFileSystem</value>
>   <description>The implementation class of the S3A Filesystem</description>
> </property>
> ```

But here it actually _does mean_ "class".

Think of a (very...) naive PySpark or Spark-SQL user.  To them, "class" is an implementation detail _on a distant part of the system_.